.circleci/config.yml

```diff
@@ -60,7 +60,7 @@ jobs:
     amd64:
         <<: *nativejob
         environment:
-            GOVER: latest
+            GOVER: 1.14
 
     # Test against older version of golang
     golang_1_12:
```

Comment:
Should we add 1_13 as an older version here?

Reply:
This case is covered by `golang_1_13` job.